shared/editor/nodes/Image.tsx

```diff
@@ -475,6 +479,30 @@ export default class Image extends SimpleImage {
           dispatch?.(tr.setSelection(new NodeSelection($pos)));
           return true;
         },
+      editLink: (): Command => (state, dispatch) => {
```

Comment:
```suggestion
      editImageUrl: (): Command => (state, dispatch) => {
```

These commands are "global" to the editor, so should be more specific

Reply:
I would keep it specific –  it's often a different attribute anyway, e.g. `url`, `src`, `href`